src/olympia/addons/tests/test_update.py

```diff
@@ -91,12 +91,14 @@ def test_no_version(self):
         assert up.is_valid()
 
     def test_unlisted_addon(self):
-        """Don't provide updates for unlisted addons."""
+        """Add-ons with only unlisted versions are valid, they just don't
```

Comment:
Ah, I misunderstood :) I don't think it matters much, the code serves a different rdf if there are no versions. 

There is a related issue that's more worrying though: https://github.com/mozilla/addons-server/issues/3228 causes `is_valid()` to return `True` if the addon status is `STATUS_DISABLED`. I should change that, but in a different PR.